app/views/home.html

```diff
@@ -1,12 +1,18 @@
-<h1>Pokédex Tracker</h1>
+<div class="home">
+  <div class="hero">
+    <img src="/pokeball.svg">
+    <h1>Pokédex Tracker</h1>
+  </div>
+
+  <div class="sub">
+    <h2>A tool for tracking your <a href="http://bulbapedia.bulbagarden.net/wiki/Living_Pok%C3%A9dex" target="_blank" class="link">Living Dex</a> progress. Easily toggle between and track your captured Pokémon, find the locations of those left to be captured, and share a public link with others to see how you can help each other out. Check out an example living dex <a class="link" href="https://pokedextracker.com/u/ashketchum10">here</a>!</h2>
```

Comment:
for the example link, can you do `<a [routerLink]="['Tracker', { username: 'ashketchum10' }]">` so it uses the angular router?